foundations/foundation-common/src/main/java/io/servicecomb/foundation/common/net/NetUtils.java

```
@@ -151,18 +149,18 @@ public static IpPort parseIpPortFromURI(String uriAddress) {
    */
   public static String getRealListenAddress(String schema, String address) {
     if (address == null) {
-      return address;
+      return null;
```

It could be better to throw the exception out.